test/plugin/scenarios/resteasy-4.x-scenario/support-version.list

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+4.7.6.Final
```

Comment:
We need more major versions of 4.x. It is better one for 4.1, 4.2, 4.3, etc.

Reply:
so, I need to add more version in this file, and modify `resteasy.version` to `${test.framework.version}` in resteasy-4.x-scenario's `pom.xml` , and run tests again ？